contracts/batcher/Batcher.sol

```diff
@@ -27,6 +27,18 @@ abstract contract Batcher is IBatcher, UOwnable {
         __UOwnable__initialize();
     }
 
+    function reserve() external view returns (address) {
```

Comment:
We don't need these extra view functions if we just update the interface to return the `public` variables above